src/algokit/core/version_prompt.py

```diff
@@ -44,7 +44,7 @@ def get_latest_version_or_cached() -> str | None:
     version_check_path = get_app_state_dir() / "last-version-check"
 
     try:
-        last_checked = os.path.getmtime(version_check_path)
+        last_checked = Path.stat(version_check_path).st_mtime
```

Comment:
since `version_check_path` is already a `Path` object, this should just be `version_check_path.stat().st_mtime`